src/SFA.DAS.Assessor.Functions/Domain/Print/DeliveryNotificationCommand.cs

```diff
@@ -76,8 +76,7 @@ private async Task ProcessFile(string fileName)
                 Status = n.Status,
                 StatusDate = n.StatusChangeDate
             }));
-
-            _fileTransferClient.DeleteFile($"{_sftpSettings.DeliveryNotificationDirectory}/{fileName}");
+            _fileTransferClient.MoveFileToArchive($"{_sftpSettings.DeliveryNotificationDirectory}/{fileName}", _sftpSettings.ArchiveDeliveryNotificationDirectory);
```

Comment:
This should be MoveFile as you are specify the from and to - in your test it should test that it calls move for a file in the delivery notification to the archive folder.